package.json

```diff
@@ -67,6 +66,7 @@
     "npm-package-arg": "^11.0.2",
     "npm-registry-fetch": "^16.2.1",
     "rimraf": "^5.0.5",
+    "semver": "^7.6.0",
```

Comment:
`semver` is a CommonJS module and a dependency of both `taze` and `taze`'s dependency.

Rollup has trouble handling a CommonJS external from a transitive dependency, so let's bundle `semver` as well.